### PR TITLE
chore: remove sm studio module

### PR DESCRIPTION
**Pull request by kukushking**
*Description of changes:*
- removed SageMaker Studio module that now moved to [MLOps Modules](https://github.com/awslabs/mlops-modules)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: IDFModuleTestsInfrastructur-59JoFoVH64CI
* Commit ID: 6155dca3af2fd97cfe96b3c4e5268d1c7700bce5
* Result: SUCCEEDED
* [Build Logs](https://mz6i2qapub.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=04d5b766-001e-4407-bd39-1fb1746fc154%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,6 +18,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### **Removed**
 
+- removed SageMaker Studio module that moved to [MLOps Modules](https://github.com/awslabs/mlops-modules)
```
Comment (dgraeber) [inline CHANGELOG.md:21]: excellent

**Review by dgraeber — DISMISSED** [review]

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: IDFModuleTestsInfrastructur-59JoFoVH64CI
* Commit ID: b1393b0eac682afafe35febf2c4c682da44b1139
* Result: SUCCEEDED
* [Build Logs](https://mz6i2qapub.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=ddf1ef76-9692-47e7-af32-0ef31e840ef5%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by kukushking** [comment]
@dgraeber updated the manifests, too

**Review by srinivasreddych — APPROVED** [review]

**Pull request merged**